doc/src/region.rst

```diff
@@ -203,12 +204,13 @@ and with radius as its radius.
 
 The *radius* value for styles *sphere* and *cylinder*, and the
 parameters a,b,c for style *ellipsoid*, can each be specified as an
-equal-style :doc:`variable <variable>`.  Likewise, for style *sphere*
```

Comment:
For better readability for the documentation source, there should be two blanks after a full stop. We usually add them when revising documentation. There is a lot of legacy text without. Removing those extra blanks just creates extra work for the folks maintaining the manual.